app/components/SummaryList.js

```diff
@@ -0,0 +1,13 @@
+export const SummaryListItem = ({ label, value }) => (
+  <div className="summary-list__row">
+    <dt className="summary-list__key">{label}</dt>
+    <dd className="summary-list__value">{value}</dd>
+  </div>
+);
+
+export const SummaryList = ({ title, children }) => (
+  <div className="card">
```

Comment:
```suggestion
  <div className="card card--container">
```

Reply:
We might also want to give ourselves the flexibility to add other classNames to this for layout.

Reply:
Yeah this is the philosophical debate about whether or not we want the card to be part of this component or not. I'm inclined to say that if we start getting into situations where we need to pass in a className to the card, then maybe it shouldn't be part of the SummaryList component?